### PR TITLE
implement config_from_module

### PR DESCRIPTION
**Pull request by leifurhauks**

### CODE REVIEW

File: goblin/driver/cluster.py [hunk 1]
```diff
@@ -165,8 +166,15 @@ def _get_message_serializer(self, config):
...
         return config
 
-    def config_from_module(self, filename):
-        raise NotImplementedError
+    def config_from_module(self, module):
+        if isinstance(module, str):
+            module = importlib.import_module(module)
+        config = dict()
+        for item in dir(module):
+            if not item.startswith('_') and item.lower() in self.DEFAULT_CONFIG:
```
Comment (davebshow) [inline goblin/driver/cluster.py:174]: Just curious, why do you check `not item.startswith('_')`? Can something start with `_` and be in `self.DEFAULT_CONFIG`?

**Comment by davebshow** [comment]
@leifurhauks this looks great, just the one question

File: goblin/driver/cluster.py (hunk 1 above)
Comment (leifurhauks) [reply goblin/driver/cluster.py:174]: It just short-circuits the expression, so it won't transform the string and check for membership in DEFAULT_CONFIG at all if it starts with '_'. Admittedly it is a bit nitpicky, the performance savings is likely inconsequential.

**Comment by davebshow** [comment]
There is something funny about how the config imports work in general I think.

File: goblin/driver/cluster.py (hunk 1 above)
Comment (davebshow) [reply goblin/driver/cluster.py:174]: Yeah ok makes sense

**Pull request merged**